config/300-crds/300-taskrun.yaml

```diff
@@ -1954,6 +1954,9 @@ spec:
                           type: boolean
                         disableInlineSpec:
                           type: string
+                        enable-step-actions:
+                          description: EnableStepActions is a no-op flag since StepActions are stable
+                          type: boolean
```

Comment:
Why the name change? The other flags use camel case - we should probably keep the original name.

Reply:
changed back to the old name, I removed everything related to the flag and added the flag back again which caused this, but have reinstated the old key. thank you for the second set of eyes 🙇